pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -5,16 +5,42 @@ import (
 )
 
 const (
-	// SecurityContextConstraintPrivileged is used as the name for the system default privileged scc.
+	// SecurityContextConstraintPrivileged allows access to all privileged and host features and the ability
+	// to run as any user, any group, any fsGroup, and with any SELinux context.
+	//
+	// WARNING: this is the most relaxed SCC and should be used only for cluster administration.
+	// Grant with caution.
 	SecurityContextConstraintPrivileged = "privileged"
-	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
+	// SecurityContextConstraintRestricted denys access to all host features and requires pods
+	// to be run with a UID, SELinux context, fs group, and supplemental groups that are
+	// allocated to the namespace.  This is the most restrictive SCC.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot provides all features of the SecurityContextConstraintRestricted
+	// SCC but allows users to run with any non-root UID.  The user must specify the UID or it
+	// must be specified on the by the manifest of the container runtime.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount provides all the features of SecurityContextConstraintRestricted
+	// but allows host mounts by a pod.  This is primarily used by the persistent volume recycler.
+	//
+	// WARNING: this SCC allows host file system access.  Grant with caution.
+	SecurityContextConstraintHostMount = "hostmount"
+	// SecurityContextConstraintHostNS allows access to all host namespaces but still requires pods
+	// to be run with a UID, SELinux context, fs group, and supplemental groups that are
+	// allocated to the namespace.
+	//
+	// WARNING: this SCC allows host access to namespaces, file systems, and PIDS.  It should only
```

Comment:
By descriptions I meant annotations on the SCCs themselves (kubernetes.io/description).  Doesn't do much good here for end users.